TextToSpeech/impl/TTSSpeaker.cpp

```diff
@@ -454,6 +493,13 @@ void TTSSpeaker::createPipeline() {
         result = gst_element_link_many (m_source,capsfilter,m_audioSink,NULL);
     }
 #elif defined(PLATFORM_AMLOGIC)
+    gst_bin_add_many(GST_BIN(m_pipeline), m_source, parser, decodebin, convert, resample, m_audioSink, NULL);
```

Comment:
Why is this AMLOGIC specific implementation change required?

Reply:
The AMLOGIC code was already there and not made by me